crates/ra_ide_api/src/lib.rs

```diff
@@ -60,9 +61,10 @@ pub use crate::{
     references::ReferenceSearchResult,
     assists::{Assist, AssistId},
     hover::{HoverResult},
+    folding_ranges::{folding_ranges, Fold, FoldKind},
```

Comment:
`folding_ranges` function should be `pub(crate)` I think

Reply:
my bad